taxonomies/ingredients.txt

```diff
@@ -8841,17 +8892,101 @@ wikipedia:en:https://en.wikipedia.org/wiki/Xylanase
 # usage:fr:xylanase (contient blé)
 # usage:fr:enzymes (xylanases)
 
-
-en:dairy, dairy product
-fr:produit laitier
-vegan:en:no
-
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 #
-#                                           B U T T E R     S E C T I O N
+#                               D A I R Y     P R O D U C T S     S E C T I O N
```

Comment:
Note that these are not products, but ingredients. Products are found in the categories.

Reply:
Sure, but nevertheless, all milk products, categorised as ```en:dairy``` as main category.
Here I just made some improvements on the headings, which are just comments … Now it's as following:
```
# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
#
#                               D A I R Y     P R O D U C T S     S E C T I O N
#
# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
################################ BUTTER related stuff ##########################
################################## MILK related stuff ##########################
########################## LOW FAT and SKIMMED MILK ############################
################################# SEMI-SKIMMED MILK ############################
################################## FULL FAT MILKS ##############################
################################## RAW MILKS ###################################
################################## COW MILKS ###################################
################################# GOAT MILKS ###################################
################################# SHEEP MILKS ##################################
################################# BUFFALO MILKS ################################
################################# CONCENTRATED MILKS ###########################
################################ MILK POWDERS ##################################
######################### MILK and CREAM related stuff #########################
###################### LACTOSE and MINERALS related stuff ######################
################################## YOGURT related stuff ########################
################################## KEFIR related stuff #########################
################################## WHEY related stuff ##########################
################################## CREAM related stuff #########################
############################## SOURED MILK related stuff #######################
############################### BUTTERMILK related stuff #######################
```

Reply:
Was just to remind you. And indeed many ingredients can also be found as products in the categories taxonomy.

Reply:
And en:dairy is actually found as ingredient, so we better support it well.

Reply:
Okay, thanks!